nservicebus/hosting/cloud-services-host/hosting-options.md

```diff
@@ -1,84 +0,0 @@
----
```

Comment:
yep that is fine. just want to make sure if someone is linking to this from an external source they will get a 404 and that is what we want